src/components/Details/VmDetails/VmDetails.js

```diff
@@ -0,0 +1,158 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { has } from 'lodash';
+import { VmStatus } from '../../VmStatus';
+import {
+  getCpu,
+  getDescription,
+  getFlavor,
+  getMemory,
+  getNodeName,
+  getOperatingSystem,
+  getVmTemplate,
+  getWorkloadProfile,
+} from '../../../utils';
+import { CUSTOM_FLAVOR } from '../../../constants';
+
+const DASHES = '---';
+
+export const getVmIpAddresses = vmi => {
+  let ipAddresses = [];
+  if (has(vmi, 'status.interfaces')) {
+    ipAddresses = vmi.status.interfaces.map(iface => iface.ipAddress);
+  }
+
+  return ipAddresses;
+};
+
+const Flavor = props => {
+  const { vm } = props;
+  const flavor = getFlavor(vm);
+  if (!flavor) {
+    return DASHES;
+  }
+  const isCustomFlavor = flavor === CUSTOM_FLAVOR;
+
+  let resourceElement = null;
+  if (isCustomFlavor) {
+    const cpu = getCpu(vm);
+    const memory = getMemory(vm);
+    let resourceStr = '';
+    resourceStr += cpu ? `${cpu} CPU` : '';
+    resourceStr += memory ? `, ${memory} Memory` : '';
+    resourceElement = <div>{resourceStr}</div>;
+  }
+
+  return (
+    <Fragment>
+      <div>{flavor}</div>
+      {resourceElement}
+    </Fragment>
+  );
+};
+
+Flavor.propTypes = {
+  vm: PropTypes.object.isRequired,
+};
+
+export const VmDetails = props => {
+  const { launcherPod, importerPod, migration, NodeLink, vm, vmi, PodResourceLink, NamespaceResourceLink } = props;
+  const nodeName = getNodeName(launcherPod);
+  const description = getDescription(vm);
+  const ipAddresses = getVmIpAddresses(vmi);
+
+  return (
+    <Fragment>
+      <div className="co-m-pane__body">
+        <h1 className="co-m-pane__heading">Virtual Machine Overview</h1>
+        <div>
+          <div className="row">
```

Comment:
We could use pf-react (bootstrap) `Row` and `Col`  components here. I  personally do not mind this approach but we are already using `Col` in this project, so I guess it would be better to be consistent.